coordinator/src/main/java/org/apache/uniffle/coordinator/CoordinatorGrpcService.java

```diff
@@ -545,4 +548,17 @@ private CoordinatorRpcAuditContext createAuditContext(String command) {
     }
     return auditContext;
   }
+
+  @Override
+  public void update(RssConf conf, Map<String, Object> changedProperties) {
+    if (changedProperties == null || conf == null) {
```

Comment:
Actually, I don't hope the other change properties are exposed to independent update method, the listener should only pay attention to its properties. If not updated, the update should be skipped.

So the suggestion is like as follows:

1. register the attentioned key and update callback to the ReconfigurationRegistry.
2. When the attentioned key is changed, then to invoke the callback by the ReconfigurationManager. 

WDYT? @maobaolong

Reply:
Done.

1. Support register `attentioned key`
2. Change to use callback function instead of implements ReconfigureListener.